### PR TITLE
Whos on first

### LINKED ISSUE #51: Add concise list of what Search can do
See the discussion on this PR: https://github.com/pelias/pelias-doc/pull/45

Determine if it belongs in the get started, index, or both based on concurrent work to make index page better for docs site.

### LINKED ISSUE #80: Specify possible layers in Reverse Geocoding page
https://github.com/mapzen/mapzen-docs/blob/master/search/reverse.md
May be listed elsewhere, but not listed here.

### LINKED ISSUE #89: Update docs to reflect changes for Who's on First dataset
Update existing topics on data sources, endpoints with sources, place, etc. to cover quattroshapes and geonames changes

Add new content about WoF - including description, adding geometry, bounding box polygons for search and autocomplete

Add what's new section with deprecation notice + new source

### PR DESCRIPTION
**Pull request by riordan**
No seriously, who?

Adds all new changes for Who's on First milestone.

fixes #51
fixes #89
fixes #80

### CODE REVIEW

**Comment by riordan** [comment]
Missing Retext sanitization.

**Comment by rmglennon** [comment]
fixes #51 
fixes #89

**Comment by riordan** [comment]
This needs a better review of new ID standards.

File: data-sources.md [hunk 1]
```diff
@@ -2,42 +2,88 @@
...
 
-[Quattroshapes](http://quattroshapes.com/) provides global coverage of location data for:
+OpenAddresses is by far the largest dataset by number of records used by Mapzen Search. Even though it only contains address data (as in no building names or other metadata), it's a great resource for global geocoding.
 
-- countries
-- regions (states/provinces)
-- counties
-- localities (cities, towns, hamlets, villages)
-- neighborhoods (in many places)
+Many of the sources used in OpenAddresses require [attribution (LINK NOT LIVE)](https://www.mapzen.com/copyright/openaddresses). Mapzen Search _does not_ import OpenAddresses sources marked as Share-Alike by the OpenAddresses project.
```
Comment (orangejulius) [inline data-sources.md]: ooh, we have to remember to change this before we push the button :)

**Comment by orangejulius** [comment]
:+1: looks good to me!

File: data-sources.md [hunk 2]
```diff
@@ -2,42 +2,92 @@
...
 
-Quattroshapes data is licensed [CC BY](http://creativecommons.org/licenses/by/2.0/), allowing its use for any purpose with proper attribution.
+Layers:
+- `country`
+- `macroregion`
+- `region`
+- `macrocounty`
+- `county`
+- `locality`
+- `localadmin`
```
Comment (dianashk) [inline data-sources.md]: missing `borough`

**Pull request merged**